flutterPractice04/main.dart

```diff
@@ -0,0 +1,12 @@
+int sumNumbersInArray(var arr) {
+
+  var result = 0;
+    for(var i = 0;i<arr.length;i++){
```

Comment:
Please add spaces after each `;`. Btw, Visual Studio Code can format the whole file for you.